app/app.go

```diff
@@ -128,7 +129,7 @@ import (
 )
 
 const (
-	Name       = "centauri"
+	Name       = "pica"
 	dirName    = "banksy"
```

Comment:
why not chnge dirname?

Reply:
I thought it's not necessary because normal user never see it. And it also make another complicate step for validator

Reply:
you mean `mv -r ~/.banksy ~/.cenauri` is complicated? how so?

Reply:
@dzmitry-lahoda in the case validator use tool like `cosmosia` or `cosmosvisor`, this change will break the automate process. I suggest we should not make this change because it's not effective with the chain affect the chain functionality.

Reply:
i see.

but daemon name has all same problems and does breake atuomate process and its. ot effective the chain functionality 

https://docs.cosmos.network/main/build/tooling/cosmovisor#command-line-arguments-and-environment-variables

Reply:
i guess same could work for cosmosia. 

but sure. let keep it.

but not sure than why rename binary name as it does not effective with the chain functionality.

Reply:
@dzmitry-lahoda as i understand now
this PR does not contains **anymore** changes related to prefix change from `cetnauri` to `pica`
PR contains only changes to `key rotation`.
What this key rotation suppose todo according to prefix changes
Seems that we need to close this pr if we do not any key rotation and i will push changes from other branch.